spotbugsTestCases/src/java/PotentialSecurityCheckBasedOnUntrustedSource.java

```diff
@@ -0,0 +1,39 @@
+import java.io.File;
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+
+public class PotentialSecurityCheckBasedOnUntrustedSource {
```

Comment:
If possible, could you add more test cases to test with lambda?
The `PrivilagedAction` is a functional interface, so coder may implement like: `AccessController.doPrivileged((Object object) -> { ... })`?